sympy/sets/fancysets.py

```diff
@@ -1338,6 +1345,8 @@ def _contains(self, other):
         isTuple = isinstance(other, Tuple)
         if isTuple and len(other) != 2:
             raise ValueError('expecting Tuple of length 2')
+        if other is S.ComplexInfinity:
+            return False
```

Comment:
I think it should be tested if either `None` or `nan` could be used. There does not seem to be other objects that could be thought of as 'indeterminate'. That would then also solve the problem of `zoo` in `Complexes` with no need to change the latter.